config/config.go

```diff
@@ -464,6 +464,16 @@ type Config struct {
 
 	// Secrets are key-value pairs that can be accessed in the dashboard via "secrets://"
 	Secrets map[string]string `json:"secrets"`
+
+	// TykErrors is used to override returned API error codes and messages.
+	TykErrors map[TykErrorType]TykError `bson:"tyk_errors" json:"tyk_errors"`
```

Comment:
Maybe align it with what we already have in python plugins, and name it `override_errors`? 
Note that in future we can add a notation to override upstream server errors too.